stdlib/LinearAlgebra/src/bunchkaufman.jl

```diff
@@ -4,6 +4,37 @@
 ## LD for BunchKaufman, UL for CholeskyDense, LU for LUDense and
 ## define size methods for Factorization types using it.
 
+"""
+    BunchKaufman <: Factorization
+
+Matrix factorization type of the Bunch-Kaufman factorization of a `Symmetric` or
```

Comment:
Can we x-ref `Symmetric` and/or `Hermitian` here?

Reply:
Sure.

Reply:
Actually, I realized that this may be misleading. What you need is that the matrix is numerically symmetric or Hermitian, it doesn't have to be of `Symmetric` or `Hermitian` type. I'd suggest to type these expressions as regular text rather than as `code`. That's how it's done in cholesky as well.